styx-scheduler-service/src/main/java/com/spotify/styx/docker/KubernetesDockerRunner.java

```diff
@@ -677,8 +677,6 @@ private void emitPodEvents(Pod pod, RunState runState) {
       }
 
       try {
-        // TODO: spoofing counter values like this can give unexpected results, e.g. if we emit two events here the
```

Comment:
This comment has been addressed because if the first event gets rejected, the whole method would return without further trying.